lib/expressapp.js

```diff
@@ -27,9 +27,10 @@ ExpressApp.start = function(opts) {
   WalletService.initialize(opts.WalletService);
   var app = express();
   app.use(function(req, res, next) {
-    res.setHeader('Access-Control-Allow-Origin', '*');
+    res.setHeader('Access-Control-Allow-Origin', req.headers.origin);
+    res.setHeader('Access-Control-Allow-Credentials', 'true');
```

Comment:
I'm not so sure about this. From [this answer on stackoverflow](http://stackoverflow.com/questions/24687313/what-exactly-does-the-access-control-allow-credentials-header-do), this header is used to accept cookies in cross-site requests. Do we even need this in BWS?